src/routes/api/client-key/client-key-regex.test.ts

```diff
@@ -1,9 +1,11 @@
 import { extractClientKey } from "./client-key-regex";
 
 const sampleUuid = "yyyyyyyy-0569-4f33-8413-cccccccc";
+const sampleJiraSite = "sample-site.atlassian.net";
```

Comment:
Some of the jira site does have a client key as domain name.

Reply:
:wat:

Reply:
🤷

Reply:
I mean, you could add anything as a site name right? :P

My guess is some kind of import from server to cloud?

Reply:
`import from server to cloud?` make sense, that could be the reason.
From app's point of view, it shouldn't matter, it is just a String. So I think it is okay.